protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/utils.ts

```diff
@@ -325,3 +326,35 @@ export const getSaveStepSnackbarText = (
 
 export const capitalizeFirstLetter = (stepName: string): string =>
   `${stepName.charAt(0).toUpperCase()}${stepName.slice(1)}`
+
+type ErrorMappedToField = Record<string, FormError>
+
+export const getFormErrorsMappedToField = (
+  formErrors: StepFormErrors
+): ErrorMappedToField => {
+  return formErrors.reduce<ErrorMappedToField>((acc, error) => {
+    const { dependentFields } = error
+    for (const field of dependentFields) {
+      acc[field] = {
+        ...error,
+        showAtField: error.showAtField ?? true,
+        showAtForm: error.showAtForm ?? true,
```

Comment:
i think this makes sense to both default to true. so if a new error is created but the showAtField/showAtForm isn't wired up, it'll show up in both places in the meantime. Do you mind adding a console.error as well in here? Something like the following:

The console.error is just extra assurance that people adding new errors in the future will wire up these fields.

```suggestion
  return formErrors.reduce<ErrorMappedToField>((acc, error) => {
    const { dependentFields } = error
    
    for (const field of dependentFields) {
    
    if (error.showAtField == null && error.showAtForm == null) {
    console.error(`${error.title}` needs to wire up where the error is shown`)
    }
    
      acc[field] = {
        ...error,
        showAtField: error.showAtField ?? true,
        showAtForm: error.showAtForm ?? true,
```